app/views/content/github/_review_checklist.erb

```diff
@@ -10,8 +10,7 @@
 
 - [ ] **Repository:** Is the source code for this software available at the <%= link_to "repository url", paper.repository_url, :target => "_blank" %>?
 - [ ] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
-- [ ] **Version:** Does the release version given match the GitHub release (<%= paper.software_version %>)?
-- [ ] **Authorship:** Has the submitting author (<%= paper.submitting_author.github_username %>) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
+- [ ] **Contribution and authorship:** Based on the list of <%= link_to "contributors", paper.repository_url + "/graphs/contributors", :target => "_blank" %> to this project, can you confirm: 1) that the submitting author (<%= paper.submitting_author.github_username %>) made major contributions to the software?, and 2) That the full list of paper authors seems appropriate and complete?
```

Comment:
So my only problem with this is that the link will only work if the source code is on GitHub. We _prefer_ submissions that are on GitHub and stored in git but adding URLs like this makes this even more locked in. Thoughts?

Reply:
I see the problem. Did not think about that. We can remove the URL so.